pkg/network/ebpf/c/prebuilt/shared-libraries.c

```diff
@@ -0,0 +1,12 @@
+#include "kconfig.h"
+#include "bpf_tracing.h"
+#include "bpf_telemetry.h"
+#include "bpf_builtins.h"
+
+#include <uapi/linux/ptrace.h>
+
+#include "shared-libraries/types.h"
+#include "shared-libraries/maps.h"
+#include "shared-libraries/probes.h"
+
+char _license[] SEC("license") = "GPL"; // NOLINT(bugprone-reserved-identifier)
```

Comment:
You can leave off this comment. I added it when I was using CLion, which I'm no longer doing (and I don't think anyone else is either).
```suggestion
char _license[] SEC("license") = "GPL";
```